docs/user/README.md

```diff
@@ -467,8 +470,8 @@ Fold the function with the token containing letter 'a' with a blue hat.
 
 Extracts a target as a variable using the VSCode refactor action
 
-- `"extract"`
-- `"extract {TARGET} as hello world"`
+- `"extract <TARGET>"`
+- `"extract  <TARGET> as hello world"`
```

Comment:
Does this one still work? I think we removed this form a while back

Reply:
extract T as hello world doesn't seem to work for me. Extract T still does

Reply:
Not sure how to verify for certain but extract ___ as ___ doesn't seem to exist.

Reply:
I've verified it by reading the ExtractVariable action file. l'll update the read me